modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -271,6 +271,13 @@ public static boolean isSupported(ConditionalFeature feature) {
         return PlatformImpl.isSupported(feature);
     }
 
+    /**
+     * The maximum number of nested event loops.
+     *
+     * @since 25
+     */
+    public static final int MAX_NESTED_EVENT_LOOPS = 200;
```

Comment:
I would rather not make this constant public.  This is essentially a safeguard measure, with the main purpose to prevent the crash and give the indication to the application developer that their program has an issue.  From this perspective, the actual number is not important and should not be codified as a public API (but definitely declared as a constant internally).

What do you think?

Reply:
I don't think it needs to be public. If we privatize this constant I could use some guidance on where to place it so that the system test can access it. Currently there are two system tests that exercise nested event loops (PlatformTest.java and NestedEventLoopTest.java) and they only rely on public API's.

Reply:
`PlatformImpl` is probably a good choice, and it's already configured to be accessible to the system tests via `tests/system/src/test/addExports`